README.md

```diff
@@ -2,4 +2,481 @@
+  * `NSString *racecar = @"racecar";`
+2. Now, create a new `BOOL` variable called `racecarIsPalindrome` and use it to capture the return of calling `stringIsPalindrome:` on `self` with `racecar` submitted as the method argument:
+  * `BOOL racecarIsPalindrome = [self stringIsPalindrome:racecar];`
+3. Finally, `NSLog()` the variables together to view the result:
+  * `NSLog(@"%d : %@", racecarIsPalindrome, racecar);`
+
+This should print: `1 : racecar`, meaning that "*racecar*" **is** a palindrome.
+
+Now, let's check a string that **isn't** a palindrome. Reuse the `palindrome` string to verify that `stringIsPalindrome:` returns "NO" for a string that doesn't match its reverse.
+
+1. Create a new `BOOL` variable called `palindromeIsPalindrome` and use it to capture the return of calling `stringIsPalindrome:` on `self` with `palindrome` submitted as the method argument:
+  * `BOOL palindromeIsPalindrome = [self stringIsPalindrome:palindrome];`
+2. Now, `NSLog()` these variables together to view the result:
+  * `NSLog(@"%d : %@", palindromeIsPalindrome, palindrome);`
+
+This should print: `0 : palindrome`, meaning that "*palindrome*" is **not** a palindrome (`BOOL ironic = YES;`).
+
+## Adding Complexity
+
+### VII. Ignoring Typecase
+
+Let's improve our `stringIsPalindrome:` method to allow it to handle uppercase letters in strings. Right now, evaluating the actual palindromes "*Bob*", "*Kanakanak*", and "*Aibohphobia*" will all return "NO" because they contain a capital letter. Verify this by adding these checks into the `application:didFinishLaunchingWithOptions:` method in the same style as evaluating "*racecar*":
+
+**Note:** *We suggest you get the practice of typing these lines out manually. Use the code block below as a guide without copy/pasting from it.*
+
+```objc
+// within application:didFinishLaunchingWithOptions:
+
+// single word, with uppercase
+NSString *bob = @"Bob";
+BOOL bobIsPalindrome = [self stringIsPalindrome:bob];
+NSLog(@"%d : %@", bobIsPalindrome, bob);
+    
+NSString *kanakanak = @"Kanakanak";
+BOOL kanakanakIsPalindrome = [self stringIsPalindrome:kanakanak];
+NSLog(@"%d : %@", kanakanakIsPalindrome, kanakanak);
+    
+NSString *aibohphobia = @"Aibohphobia";
+BOOL aibohphobiaIsPalindrome = [self stringIsPalindrome:aibohphobia];
+NSLog(@"%d : %@", aibohphobiaIsPalindrome, aibohphobia);
+```
+Running the program now should print these three lines with zeroes:
+
+```
+0 : Bob
+0 : Kanakanak
+0 : Aibohphobia
+```
+The implementation of handling these additional cases in `stringIsPalindrome:` is relatively simple: we just need to lowercase the `string` argument before we evaluate it, right? Let's try it:
+
+1 — Insert a statement that reassigns to `string` the result of calling the `lowercase` method on it:
+
+```objc
+- (BOOL)stringIsPalindrome:(NSString *)string {
+    
+    NSString *reverse = [self stringByReversingString:string];
+    
+    string = [string lowercaseString];
```

Comment:
ok. i did reassign `string` in the last section when removing punctuation. do you want me to change it there too?